src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -46,8 +49,18 @@ public async Task<ActionResult> Details(Guid id)
             {
                 AatfData aatf = await client.SendAsync(User.GetAccessToken(), new GetAatfById(id));
                 AatfContactData contactData = await client.SendAsync(User.GetAccessToken(), new GetAatfContact(id));
+                OrganisationData organisationData = await client.SendAsync(User.GetAccessToken(), new GetOrganisationInfo(aatf.Operator.OrganisationId));
 
-                AatfDetailsViewModel viewModel = mapper.Map<AatfDetailsViewModel>(new AatfDataToAatfDetailsViewModelMapTransfer(aatf, contactData));
+                List<AatfDataList> associatedAatfs = await client.SendAsync(User.GetAccessToken(), new GetAatfsByOperatorId(aatf.Operator.Id));
+
+                List<Core.Scheme.SchemeData> associatedSchemes = await client.SendAsync(User.GetAccessToken(), new GetSchemesByOrganisationId(aatf.Operator.OrganisationId));
+
+                AatfDetailsViewModel viewModel = mapper.Map<AatfDetailsViewModel>(new AatfDataToAatfDetailsViewModelMapTransfer(aatf, contactData, organisationData)
```

Comment:
Create unit tests to check that the mapper is called with the correct parameters and that the returned view model is the model returned from the mapper